mantidproject/mantid
# Fix QtSupport warnings for group 4

This is group 4 of #16128.

This fixes the Qt3Support warnings for group 4.
## To test:

Review the code.

The changes in this PR have touched many areas of MantidPlot.  Below is a list of items which should be checked. Areas that need thorough testing:
- Marker in Graphs (I think that is the cursor)
- Interface Menu
  - Check that it opens and that the options and sub options are there
- Window Menu
  - Check that it opens and that the options and sub options are there
- File Menu
  - check especially the recent files and projects
- Matrix Menu
  - Check that it opens and that the options and sub options are there
  - You need to create a Matrix and have it in focus to see it
- Analysis Menu
  - Check that it opens and that the options and sub options are there
  - I think you need to set "Enable QtiPlot fitting" in Preferences->Fitting to see this; you also need to have a plot/matrix/ in focus
- Copy and paste in Note; 
- Draw an arrow in plot
  - There is an option in the menu bar which should be enabled for plots

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
